src/org/sosy_lab/java_smt/api/FormulaType.java

```diff
@@ -270,12 +344,12 @@ public boolean equals(Object obj) {
 
     @Override
     public String toString() {
-      return "FloatingPoint<exp=" + exponentSize + ",mant=" + mantissaSize + ">";
+      return "FloatingPoint<exp=" + exponentSize + ",mant=" + getMantissaSizeWithSignBit() + ">";
     }
 
     @Override
     public String toSMTLIBString() {
-      return "(_ FloatingPoint " + exponentSize + " " + mantissaSize + ")";
+      return "(_ FloatingPoint " + exponentSize + " " + getMantissaSizeWithSignBit() + ")";
```

Comment:
Hm, this seems to be a desired behavior change of this MR, isn't it?

So it would mean that there are not just API improvements but also an actual bug fix. Hopefully this method is rarely used?

Reply:
The SMTLIB2 standard says that our previous implementation is wrong here, as statet above ` As a consequence, we also return this wrongly for toSMTLIBString() etc.`.  Example from the standard: `  -  Float32 is a synonym for (_ FloatingPoint  8  24)`. As a consequence i would change it and communicate this to the users.